src/index.css

```diff
@@ -1,27 +1,77 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  background: url('./media/math-background.jpg');
+  width: 100%;
+  height: 100%;
+}
```

Comment:
[ OPTIONAL]
- Project requiements asks for our Calculator.js component to match given design and I think matching designs always incude colors and typograpgy. Check screenshots below to see difference.
- Given design
![Screen Shot 2023-02-21 at 6 34 37 PM](https://user-images.githubusercontent.com/62849872/220506524-a026858a-152c-4c9a-8973-f36fc0a008c4.png)
- Your design
![Screen Shot 2023-02-21 at 6 34 56 PM](https://user-images.githubusercontent.com/62849872/220506571-cd693711-8f1c-47c4-8e63-d09eb398af0f.png)

Reply:
@t-yanick 
hi Cr, I hope you are doing well

as I am making this my own project I like to implement my own design well I like  to switch to the given design for approval and change it to design what I like after approval and as sir, you marked it optional sorry to say that I won't change thanks for the review
sorry to say that it is not feasible according to my thoughts